skt/runner.py

```diff
@@ -121,23 +121,23 @@ def __getxml(self, replacements):
 
         return xml
 
-    def getresultstree(self, jobid, logs=False):
+    def getresultstree(self, cid, logs=False):
         """
-        Retrieve Beaker job results in Beaker's native XML format.
+        Retrieve Beaker results for cid in Beaker's native XML format.
 
         Args:
-            jobid: The Beaker job ID.
+            cid:   ID of the job, recipe or recipe set.
```

Comment:
I believe the Beaker's official term for this is "taskspec", as described in https://beaker-project.org/docs/man/bkr.html#specifying-tasks
I might be wrong (e.g. that's just a placeholder used in that manpage only), but if it is, then we should use it here, and I was already using it through the code.
What do you think?

Reply:
TBH I don't know what the official term is and I chose one I saw used in the code already. I'm totally for changing all the `cid`s to `taskspec`s (probably in a separate pull) if that's the right term to use.